functions/src/expressRestApp/routes/route.service.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import { getData } from "../services/stations.service";
+
+const router = express.Router();
+
+router.get("/month", async (req, res) => {
+  await getData(req, res, "month");
+});

```

Comment:
currently we have only one station with name/id horiba, but in the future we going to have more, so it will be great to implement route handling for multiple stations right away:

`router.get('/:stationId/month', async (req, res, next) => {
  try {
    const stationId = req.params.stationId;
    const stationData = await getData(req, res, stationId, "month"); // getData should be updated with new argument stationId  
    res.status(200).json(stationData);
  } catch (error) {
    next(error); // Pass the error to the errorHandler middleware
  }
});`